MagmaSafe.Repositories/UserRepository.cs

```diff
@@ -13,13 +13,6 @@ namespace MagmaSafe.Repositories
     public class UserRepository : IUserRepository
     {
         private readonly IRepositoryHelper helper;
```

Comment:
Era pra remover só o secutiry helper mano kkkk 

```suggestion
        private readonly IRepositoryHelper helper;

        public UserRepository(IRepositoryHelper helper)
        {
            this.helper = helper;
        }
```